third_party/tsl/tsl/platform/cpu_info.cc

```diff
@@ -478,6 +516,7 @@ class CPUIDInfo {
   int cpunum_;
   int is_arm_neoverse_v1_;  // ARM NEOVERSE V1
   int is_arm_neoverse_n1_;  // ARM NEOVERSE N1
+  int has_bf16;
```

Comment:
Nit: Please end class member variable's names with [an underscore](https://google.github.io/styleguide/cppguide.html#Variable_Names).
```suggestion
  int has_bf16_;
```